lib/publiccloud/azure.pm

```diff
@@ -66,9 +66,9 @@ sub find_img {
     return if (!$self->resource_exist());
 
     ($name) = $name =~ m/([^\/]+)$/;
-    my $sku = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen2')) ? 'V2' : 'V1';
+    my $gen = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen2')) ? 'V2' : 'V1';
     $name =~ s/\.xz$//;
-    $name =~ s/\.vhdfixed$/-$sku.vhd/;
+    $name =~ s/\.vhdfixed$/-$gen.vhd/;
```

Comment:
```suggestion
    $name =~ s/\.vhdfixed.xz$/-$gen.vhd/;
```
will allow also drop line above ? :thinking:

Reply:
What if we decide to provide uncompressed image? I mean, it's corner case but it may happen (maybe in development or in manual yet openQA-assisted testing)?

Reply:
not that I get you point , if we will decide to do so we will need to do extra PR in both version , right ? And my suggestion not over complicate works needs to be done in such case . But it is not a big deal if you like to keep it let's keep it